scripts/notarize.js

```diff
@@ -1,41 +1,36 @@
-const fs = require("fs");
-const path = require("path");
-const { notarize } = require("electron-notarize");
-const envConfig = require('dotenv').config({path:path.join(__dirname, '../.env')});
+const fs = require('fs')
+const path = require('path')
+const { notarize } = require('electron-notarize')
+const envConfig = require('dotenv').config({ path: path.join(__dirname, '../.env') })
 
 Object.entries(envConfig.parsed || {}).forEach(([key, value]) => {
-  process.env[key] = value;
-});
-
+  process.env[key] = value
+})
 
 module.exports = async function (params) {
-
-    // Only notarize the app on Mac OS only.
-  if (process.platform !== "darwin") {
-    return;
+  // Only notarize the app on Mac OS only.
```

Comment:
By the way, JFYI we plan to notarize it on windows as well, see #2530.

Reply:
This was my first time with electron and I have no idea what it is xD